src/OutputWriters/jld2_output_writer.jl

```diff
@@ -258,18 +261,17 @@ end
 function write_output!(writer::JLD2OutputWriter, model)
 
     verbose = writer.verbose
-    path = writer.filepath
```

Comment:
why this change?

Reply:
Because `path` was used later on in the code, but it wasn't updated. Thus the code crashed while  creating the file and when the `writer.filepath` changed  but not the `path` . The easier fix was to replace `path` by the `writer.filepath`.

Reply:
Ok, that's a good reason!